src/components/dashboard/components/SideNavigation.jsx

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
+/* eslint-disable */
```

Comment:
why do we have this comment?

Reply:
Some statement that helps disable some funny rule